drake/util/drakeGeometryUtil.cpp

```diff
@@ -19,168 +19,6 @@ double angleDiff(double phi1, double phi2)
   return d;
 }
 
-
-Vector4d quatConjugate(const Eigen::Vector4d& q)
```

Comment:
> while it's nice to see all of this go, it does mean more in the headers, etc. 

Almost everything else in drakeGeometryUtil is already in the header, and I think that's where it should be for this low level stuff.

> did you evaluate performance?

No. But it should be exactly the same.

> one of the big things about all of these geometry utils is that they work on fixed-size matrices (and can potentially avoid most dynamic allocation).

Exactly.

> Is the same true for your intializeAutodDiffArgs version?

Yeah, if all of the inputs have fixed size, then the derivative vector will also have fixed size (being the sum of the sizes at compile time of all of the input arguments). Adding this to the documentation. I'd like to handle the fixed max size case too, but later.